src/gui/folderwatcher_mac.h

```diff
@@ -37,6 +37,9 @@ class FolderWatcherPrivate
     QStringList addCoalescedPaths(const QStringList &) const;
     void doNotifyParent(const QStringList &);
 
+    /// On OSX the watcher is ready when the ctor finished.
+    bool _ready = 1;
```

Comment:
I guess clang-tidy didn't complain because we don't run it on mac yet, but likely you can do the same adjustment than above and use "true" instead of "1" in that commit.